### PR TITLE
Fixes Part of #331: Moved Pre-requisites before Installation in Docker Installation

### PR DESCRIPTION
**Pull request by jayam04**
Fixed part of #331 
1. Moved Pre-requisites section above Installation section, well because it's better that way.
2. Updated E2E test commands for Docker

### CODE REVIEW

**Comment by jayam04** [comment]
@seanlip PTAL

**Comment by jayam04** [comment]
@seanlip PTAL

File: Installing-Oppia-using-Docker.md [hunk 1]
```diff
@@ -240,10 +240,11 @@ These are the `make` commands available for running tests:
 **Flags:**  
 	Usage - `make run_tests.e2e FLAG_NAME=VALUE`
 	* `suite` - The suite to run the acceptance tests
+   * `MOBILE` - Runs the mobile acceptance tests.
 
 	Usage - `make run_tests.acceptance PYTHON_ARGS="--flag_name"`
 	* `--skip-build` - Skips building files.
-	* `--prod_env` - Runs tests in production mode.
+	* `--prod_env` - Run acceptance test in mobile viewport
```
Comment (seanlip) [inline Installing-Oppia-using-Docker.md]: This doesn't make sense, was this change intended?

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @jayam04. There seems to be quite a bit more in this PR than what was stated in the description; can you update the description?

Also I left a comment, PTAL, thanks.

**Comment by jayam04** [comment]
@seanlip Updated Description and fixed suggestion. PTAL.

**Comment by jayam04** [comment]
@seanlip Can you PTAL.

**Review by seanlip — APPROVED** [review]
Sorry for the late reply! LGTM.

Also, just a note, next time please assign me if you want a review (i.e. make sure my GitHub username is under the Assignees section). Thanks!

**Comment by jayam04** [comment]
@seanlip I am aware that we should assign PR to reviewers, when review is required. I try to make sure of it, but I don't have privileges to assign people in web-docs repo.

**Comment by seanlip** [comment]
Ah sorry, I think that was a configuration issue. You should have permissions now, I think?

**Pull request merged**